syncd/NotificationProcessor.cpp

```diff
@@ -387,6 +387,7 @@ void NotificationProcessor::process_on_port_state_change(
     for (uint32_t i = 0; i < count; i++)
     {
         sai_port_oper_status_notification_t *oper_stat = &data[i];
+        sai_object_id_t rid = oper_stat->port_id;
```

Comment:
this is not needed, just use port_id directly

Reply:
ok.